test/framework/test_framework.py

```diff
@@ -267,3 +267,36 @@ def error_message(self):
         res = expdata.analysis_results()
         self.assertEqual(len(res), 0)
         self.assertEqual(expdata.analysis_status(), AnalysisStatus.CANCELLED)
+
+    @ddt.data(None, 1, 10, 100)
+    def test_job_info(self, max_experiments):
+        """Test job_info for specific backend"""
+
+        num_circuits = 10
+        class MyExp(BaseExperiment):
+            """Some arbitraty experiment"""
+
+            def __init__(self, physical_qubits):
+                super().__init__(physical_qubits)
+                
+            def circuits(self):
+                """Generate fake circuits"""
+                qc = QuantumCircuit(1)
+                qc.measure_all()
+                return num_circuits * [qc]
+
+        backend = FakeBackend(max_experiments=max_experiments)
+        exp = MyExp([0])
+
+        if max_experiments is None:
+            num_jobs = 1
+        else:
+            num_jobs = (num_circuits + max_experiments - 1) // max_experiments
+
+        job_info = {
+                "Total Number of circuits in the experiment": num_circuits,
+                "Maximum Number of circuits": max_experiments,
+                "Total Number of jobs": num_jobs
+            }
+        
+        self.assertEqual(exp.job_info(backend=backend), job_info)
```

Comment:
This test is nice, can you also add that setting the `max_circuits` experiment option affects `job_info` as expected?